### ISSUE TITLE
Alien wire icon and dropped sprite is misleading

### ISSUE THREAD

**Issue by Regalis11** [FakeFishGames/Barotrauma#14642]
### Discussed in https://github.com/FakeFishGames/Barotrauma/discussions/14620

<div type='discussions-op-text'>

<sup>Originally posted by **ubertpendragon** September 13, 2024</sup>

![image](https://github.com/user-attachments/assets/3b78131c-275e-4926-b703-70a2d2200491)
</div>

The new alien wires look like they're decorative items/structures, and it seems like a bug that they can't be resized. They're actually "real" wires used to connect devices in the wiring mode though, and they shouldn't be resizable. We should probably change their icon and dropped sprite into something closer to the loop of wire that's used on the normal wires.

**Comment by ubertpendragon** [comment]
Sorry, I think people saw that and thought they were decorative structure like the "chain" item that can be resized horizontally, when in fact what shows up is the wire _item_ and not a structure

![image](https://github.com/user-attachments/assets/8fe32d5c-4723-4222-9426-eb7615e4bc41)

They were expecting the equivalent of

```xml
  <TailFin name="" identifier="frontfchaina1" description="" scale="1" resizehorizontal="true" body="false" castshadow="false" category="Decorative">
    <sprite texture="Content/Map/SubRearBow3.png" sourcerect="280,815,744,46" depth="0.01" origin="0.5,0.5" />
    <backgroundsprite texture="Content/Map/SubRearBow3.png" sourcerect="280,815,744,46" color="60,60,60,255" depth="0.9" origin="0.5,0.5" />
  </TailFin>
```

**Comment by dinhtuananhfakefish** [comment]
fixed in 3b9bb7a236b8cfd38ca8b8df2767dcebc7e86167

**Comment by Regalis11** [comment]
Merged, closing
